src/main/resources/avro/bdg.avdl

```diff
@@ -748,299 +748,304 @@ record Variant {
 }
 
 /**
- An enumeration that describes the allele that corresponds to a genotype.
+ Allele encodings for genotypes.
  */
-enum GenotypeAllele {
+enum Allele {
```

Comment:
The suggested fix for Impala was to rewrite using strings instead.

https://issues.apache.org/jira/browse/IMPALA-2525